protocol/eth-shared-lockbox.md

```diff
@@ -35,125 +35,81 @@ OP Chains will be governed within a common Chain Cluster governance entity (the
-
-A code example would look like this:
+A minimal set of functions should include:
 
-```solidity
-function addChain(uint256 _chainId, address _systemConfig) external {
-  require(msg.sender == updater(), "Unauthorized");
-  // Add to the dependency set and check it is not already added (`add()` returns false if it already exists)
-  require(_dependencySet.add(_chainId), "Chain already added");
+- `lockETH`: Accepts ETH from the `depositTransaction` originating from a valid `OptimismPortal`.
+- `unlockETH`: Releases ETH from the `finalizeWithdrawalTransaction` originating from a valid `OptimismPortal`.
 
-  // Store the system config
-  systemConfigs[_chainId] = _systemConfig;
+Access control for `lockETH` and `unlockETH` is validated against the mapping of authorized `OptimismPortal` addresses.
 
-  // Loop through the dependency set and update the dependency for each chain
-  for (uint256 i; i < _dependencySet.length() - 1; i++) {
+The `ProxyAdmin` owner can add `OptimismPortal` addresses to the `ETHLockbox` contract. This role can be introspected in the `ETHLockbox` contract.
 
-    uint256 currentId = _dependencySet.at(i);
+### `OptimismPortal` upgrade process
 
-    // Skip recently added chain
-    if (_chainId == currentId) continue;
+A ETH migration function can be added to the `OptimismPortal` to allow transferring all ETH to the `ETHLockbox` at any time. The `ETHLockbox` address is set during initialization of the `OptimismPortal` and cannot be changed afterwards.
 
-    systemConfigs[currentId].addDependency(_chainId);
+The migration process would:
 
-    systemConfigs[_chainId].addDependency(currentId);
-  }
+- Only the `ProxyAdmin` owner can trigger the migration
+- The `OptimismPortal` MUST be first authorized in the `ETHLockbox`
+- Transfer the entire ETH balance of the `OptimismPortal` to the pre-configured `ETHLockbox`
 
-  address portal = _systemConfig.optimismPortal();
+### `ETHLockbox` merge process
 
-  // Authorize the portal on the shared lockbox
-  SHARED_LOCKBOX.authorizePortal(portal);
+The `ETHLockbox` includes a migration function to allow transferring all ETH to another existing lockbox. This enables merging liquidity pools by migrating ETH from one lockbox to another, allowing multiple portals to use the same shared liquidity.
 
-  emit ChainAdded(_chainId, _systemConfig, portal);
-}
-```
+The merge process has two parts:
 
-Note that, under the specified flow, the dependency set consistently maintains the form of a [complete graph](https://en.wikipedia.org/wiki/Complete_graph) at all times.
+1. Destination lockbox preparation:
 
-### `SharedLockbox` implementation
+- Only the `ProxyAdmin` owner can approve a specific source lockbox to send ETH
```

Comment:
There should be requirement that migration also requires the ProxyAdmin owner of the source lockbox to match the ProxyAdmin owner of the destination lockbox